scripts/generate_otlp_projection.py

```
@@ -605,19 +635,73 @@ def render_generated_test_vectors(spec: dict) -> str:
+                f"""    pub(super) fn write_{field_name}(
+        &self,
+        builder: &mut ColumnarBatchBuilder,
+        value: WireAny<'_>,
+        scratch: &mut WireScratch,
+        string_storage: StringStorage,
+    ) -> Result<(), ProjectionError> {{
+        write_wire_any_as_string(builder, self.{field_name}, value, scratch, string_storage)
+    }}"""
+            )
+        else:
+            raise ValueError(f"unknown OTLP planned field writer {writer!r}")
```

Ruff (0.15.10)

[warning] Avoid specifying long messages outside the exception class

(TRY003)